spec/openapi.yaml

```diff
@@ -822,6 +821,383 @@ paths:
+            type: string
+            example: +61555555555
+        - name: metadata_key
+          in: query
+          description: Filter results for messages that include a metadata key.
+          style: form
+          explode: true
+          schema:
+            type: string
+            example: broadcastId
+        - name: metadata_value
+          in: query
+          description: Filter results for messages that include a metadata key containing this value. If this parameter is provided, the metadata_key parameter must also be provided.
+          style: form
+          explode: true
+          schema:
+            type: string
+            example: ABC
+        - name: accounts
+          in: query
+          description: >-
+            Filter results by a specific account. By default results
+            will be returned for the account associated with the authentication
+            credentials and all sub accounts.
+          style: form
+          explode: true
+          schema:
+            type: array
+            items:
+              type: string
+        - name: status
+          in: query
+          description: >-
+            An array of message statuses
+          style: form
+          explode: true
+          schema:
+            type: array
+            items:
+              type: object
+        - name: opt_out
+          in: query
+          description: Filter results by message format.
+          style: form
+          explode: true
+          schema:
+            type: string
+            example: true
+        - name: group_by
+          in: query
+          description: Group results by a list of values, from the enumerable table above.
+          style: form
+          explode: true
+          schema:
+            type: array
+            items:
+              $ref: '#/components/schemas/group_by'
+        - name: delivery_options
+          in: query
+          description: A list of options to configure the delivery of the report.
+          style: form
+          explode: true
+          schema:
+            type: array
+            items:
+              $ref: '#/components/schemas/deliveryOptionsBody'
+      responses:
+        202:
+          description: A list of all messages received in the specified time window
+          headers: { }
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/detailresponse'
+        400:
+          description: Bad Request. Check the json response for more details on what went wrong.
+      deprecated: false
+  /v2-preview/reporting/messages/async/status:
+    get:
+      security:
+        - basic_auth: [ ]
+        - hmac_auth: [ ]
+      tags:
+        - Messaging Reports
+      summary: Get async detail report status
+      description: Retrieves the status of a detail report.
+      operationId: GetAsyncDetailStatus
+      parameters:
+        - name: report_id
+          in: query
```

Comment:
It is correct to use `query` here though as in the API blueprint we are converting you'll see:
```
+ Parameters

    + report_id: `abc` (string, required)
```
rather than the attributes on our post requests:
```
+ Request (application/json)

    + Attributes (asyncsummaryrequest)
```